File: Python/1013 - O maior.py
n = input().split()

a, b, c = int(n[0]), int(n[1]), int(n[2])

maior_AB = (a+b+abs(a-b))/2
maior_ABC = (maior_AB+c +abs(maior_AB-c))/2

print(f"{int(maior_ABC)} eh o maior")

File: Java/1013 - O maior.java
import java.io.IOException;
import java.util.Scanner;

class Main {
    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        int a = input.nextInt(); 
        int b = input.nextInt(); 
        int c = input.nextInt(); 

        int maior_ab = (a+b+ Math.abs(a-b))/2;
        int maior_abc = (maior_ab+c+ Math.abs(maior_ab-c))/2;

        System.out.println(maior_abc + " eh o maior");
    }
}